### PR TITLE
Test scripts for Skills screen

### PR DESCRIPTION
**Pull request by AshaniUmeendra**
Test scripts for Skills screen

### CODE REVIEW

File: symfony/test/functional/cypress/support/index.js [hunk 1]
```diff
@@ -18,3 +18,5 @@ import './commands'
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+require('cypress-xpath')
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/support/index.js:21]: you may need to update package.json if you need to use this dependency

**Comment by samanthajayasinghe** [comment]
pls use following format for commit messages 
{OHRM-###} : detailed-message

File: symfony/test/functional/cypress/integration/orangehrm/admin/skills.spec.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        cy.get('.oxd-input-group__message').should('include.text', 'Skill name should be unique')
+    })
+})
+//Verify Edit skill
+describe('Edit Skill', function () {
+
+    it('check edit a skill', () => {
+        cy.login(user.admin.userName, user.admin.password)
+        cy.visit('/admin/viewSkills')
+        cy.xpath('//body[1]/div[1]/div[1]/div[1]/div[2]/div[1]/div[1]/div[3]/div[1]/div[2]/div[1]/div[1]/div[4]/div[1]/button[2]/i[1]').click()
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/integration/orangehrm/admin/skills.spec.js:53]: don't use xpath like this

File: symfony/test/functional/cypress/integration/orangehrm/admin/skills.spec.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            })
+    
+})
+//Verify deleting a skill
+describe('Delete Skill', function () {
+
+    it('check delete a skill', () => {
+        cy.login(user.admin.userName, user.admin.password)
+        cy.visit('/admin/viewSkills')
+        cy.xpath('//*[@id="app"]/div[1]/div/div[2]/div/div/div[3]/div/div[2]/div[1]/div/div[4]/div/button[1]/i').click()
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/integration/orangehrm/admin/skills.spec.js:88]: same here

File: symfony/test/functional/cypress/integration/orangehrm/admin/skills.spec.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    })
+//Verify Maximum allowed charachters validation
+describe('Maximum allowed charachters validation in Skill', function () {
+
+    it('Check Maximum allowed charachters validation in Skill', () => {
+        cy.login(user.admin.userName, user.admin.password)
+        cy.visit('/admin/saveSkills')
+        cy.xpath('//*[@id="app"]/div[1]/div/div[2]/div/div/form/div[1]/div/div[2]/input').type('01234567890123456789012345678901234567890123456789')
+        cy.get('.oxd-input-group__message').should('include.text', 'Should be less than 50 characters')
+        cy.xpath('//*[@id="app"]/div[1]/div/div[2]/div/div/form/div[2]/div/div[2]/textarea').type('0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789')
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/integration/orangehrm/admin/skills.spec.js:112]: text you can move into fixture file

**Review by samanthajayasinghe — CHANGES_REQUESTED** [review]

**Comment by RajithaKumara** [comment]
Closing since #845

**Pull request closed without merging**